examples/typescript-reactflux/.jshintignore

```diff
@@ -0,0 +1 @@
+# Gruntfile.js
```

Comment:
We really don't need a file with only one line which is even commented out. Please remove it. Please remove the whole app-specific linting setup as well, we have a jshint system in place which is cross-app.